### ISSUE TITLE
Subtitle doesn't show on webpage

### ISSUE THREAD

**Issue by wenbo5565** [rstudio/blogdown#250]
Hi, I was trying to add a subtitle to my post by adding subtitle in YAML front. However, it doesn't work. Any idea? Thanks.

![image](https://user-images.githubusercontent.com/7663395/35637964-bf2388c6-0683-11e8-8b2a-de30e121e183.png)
![image](https://user-images.githubusercontent.com/7663395/35637989-d69eb5f2-0683-11e8-8e5d-466db57890e9.png)

**Comment by shrektan** [comment]
I guess it doesn't support subtitle because you are limited to use [the variables that Hugo defines](https://gohugo.io/variables/page/).

**Comment by timtrice** [comment]
@shrektan is partly correct. You can set any variable your want in the from matter and access it through Hugo. The hugo-academic them does this but by the "summary". I don't recall seeing a theme use "description", "subtitle", "abstract", etc. But it can be done. 

Regardless, this isn't a blogdown issue.

**Comment by shrektan** [comment]
It means you may use variable `description` :

+ find the _single.html_ file of your theme;
+ add `description` as kind of subtitle with the css style you like;
+ write `description: your subtitle` in the yaml file

<img width="718" alt="2018-02-01 1 55 58" src="https://user-images.githubusercontent.com/8368933/35638922-6f4138d8-06f3-11e8-8982-d0d5ce0bf910.png">

<img width="318" alt="2018-02-01 1 56 17" src="https://user-images.githubusercontent.com/8368933/35638934-77833e06-06f3-11e8-89a6-34b094be3925.png">


<img width="368" alt="2018-02-01 1 56 25" src="https://user-images.githubusercontent.com/8368933/35638944-7c7ce222-06f3-11e8-9ccb-01c7b7504a67.png">

**Comment by shrektan** [comment]
@timtrice Yes, you are right but I haven't looked into how to set up self-defined variables. I guess my example above is what you mean for.

**Comment by timtrice** [comment]
.Description is a [Page Variable] (https://gohugo.io/variables/page/#page-variables) so you're not seeing anything custom. However, OP can certainly take your route.

See [Page-Level Params](https://gohugo.io/variables/page/#page-level-params) for custom variables.

**Comment by yihui** [comment]
Whether subtitles can be displayed is completely up to the specific theme you use. On Hugo's side, you can use `.Params.subtitle` to get the `subtitle` you defined in YAML, e.g.,

https://github.com/yihui/hugo-ivy/blob/f4bd32e2cf/layouts/partials/meta.html#L3

**Comment by wenbo5565** [comment]
@yihui  I don't know hugo much. Did you mean add the meta.html file under my theme's partials folder?

**Comment by timtrice** [comment]
@wenbo5565, find me on Twitter, @timtrice. I'll walk you through it

**Comment by yihui** [comment]
@wenbo5565 Everything that I'm able to teach you has been written in the blogdown book: https://bookdown.org/yihui/blogdown/templates.html If you don't want to read it or cannot understand it, I'll just hand you over to @timtrice :)

**Comment by wenbo5565** [comment]
@yihui @timtrice  Thank you both. I'll start with the Yihui's book. If it doesn't workout, I'll reach out to Tim.
